decidim-admin/app/models/decidim/admin/abilities/admin.rb

```diff
@@ -8,6 +8,7 @@ class Admin
         include CanCan::Ability
 
         def initialize(user)
+          return unless user
```

Comment:
The `initialize` result is not used when creating an object. If we surround this code with an `if` we're actually returning `nil` anyway, so I'd rather just return in the beginning that surrounding code with multiple `if`s. 

Yes, it feels weird to me too, but since we need to define this in an `initialize` method... I can add base `Ability` class that defines the `initialize` method:

```ruby
def initialize(user)
  @user = user
  set_permissions
end

def set_permissions
  raise NotImplementedError
end
```

Classes inheriting from this base class will just need to define the `set_permissions` method. 

Do you prefer this alternative?